Code/Mantid/Framework/DataHandling/src/LoadSpiceXML2DDet.cpp

```diff
@@ -437,5 +518,150 @@ MatrixWorkspace_sptr LoadSpiceXML2DDet::createMatrixWorkspace(
   return outws;
 }
 
+//----------------------------------------------------------------------------------------------
+/** Set up sample logs from table workspace loaded where SPICE data file is
+ * loaded
+ * @brief LoadSpiceXML2DDet::setupSampleLogFromSpiceTable
+ * @param matrixws
+ * @param spicetablews
+ * @param ptnumber
+ */
```

Comment:
Could these empty parameter description produce doxygen warnings?
Apparently doxygen is not complaining about it, but I'd have thought so.